src/main/java/com/example/tradingapp/websocket/MyWebsocketHandler.java

```diff
@@ -0,0 +1,85 @@
+package com.example.tradingapp.websocket;
+
+
+import com.example.tradingapp.websocket.encoder.LoginRequestEncoder;
+import com.example.tradingapp.websocket.model.request.LoginRequest;
+import com.example.tradingapp.websocket.model.request.SubscribeArg;
+import com.example.tradingapp.websocket.model.request.SubscribeRequest;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import org.springframework.web.socket.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+@Slf4j
+public class MyWebsocketHandler implements WebSocketHandler {
+
+    private final ObjectMapper mapper = new ObjectMapper();
+
+    @Override
+    public void afterConnectionEstablished(WebSocketSession session) throws Exception {
+        log.info("afterConnectionEstablished. session: {}", session);
+        LoginRequest login = new LoginRequestEncoder().encode();
+        String loginJson = mapper.writeValueAsString(login);
+
+        log.info("Sending login: {}", loginJson);
+        session.sendMessage(new TextMessage(loginJson));
+
+    }
+
+    @Override
+    public void handleMessage(WebSocketSession session, WebSocketMessage<?> message) throws Exception {
+        log.info("handleMessage. session: {}. message: {}", session, message);
+        log.info("Payload {}", message.getPayload());
+
+        JsonNode jsonNode = mapper.readTree(message.getPayload().toString());
+
+        if (jsonNode.has("event") && "login".equals(jsonNode.get("event").asText())) {
+
+            if ("0".equals(jsonNode.get("code").asText())) {
+                var subscribeArg = SubscribeArg.builder()
+                        .instType("ANY")
+                        .channel("orders")
+                        .build();
+                List<SubscribeArg> args = new ArrayList<>();
+                args.add(subscribeArg);
+                var subscribeRequest = SubscribeRequest.builder()
+                        .op("subscribe")
+                        .args(args)
+                        .build();
+                String subscribeJson = mapper.writeValueAsString(subscribeRequest);
+                session.sendMessage(new TextMessage(subscribeJson));
```

Comment:
just a minor style concern, but we can try extracting this code to a different method and calling it here